### PR TITLE
ocrvs-2870 Form Config Settings

### PR DESCRIPTION
**Pull request by sadmananik**

### CODE REVIEW

**Comment by sadmananik** [comment]
ocrvs-2870

**Comment by sadmananik** [comment]
This PR has a dependent PR on Farajaland. https://github.com/opencrvs/opencrvs-farajaland/pull/231

**Comment by jpye-finch** [comment]
@sadmananik the Config Settings is showing with the main SideNav and TopNav.

Instead of Form Config sideNav and topNav
image

File: packages/client/src/views/SysAdmin/Config/Forms/FormConfigSettings.tsx [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+    const { showModal, modalName, introductionPage, numberOfAddresses } =
+      this.state
+
+    return (
+      <SysAdminContentWrapper>
+        <Content
+          title={intl.formatMessage(messages.settingsTitle)}
+          titleColor={'copy'}
+        >
+          <ListView
```
Comment (atiqzaman-dsi) [inline packages/client/src/views/SysAdmin/Config/Forms/FormConfigSettings.tsx:183]: Shouldn't we use ListViewSimplified

**Review by atiqzaman-dsi — CHANGES_REQUESTED** [review]

File: packages/client/src/views/SysAdmin/Config/Forms/FormConfigSettings.tsx (hunk 1 above)
Comment (jpye-finch) [reply packages/client/src/views/SysAdmin/Config/Forms/FormConfigSettings.tsx:183]: yes

Comment (sadmananik) [reply packages/client/src/views/SysAdmin/Config/Forms/FormConfigSettings.tsx]: @atiqzaman-dsi  @jpye-finch  I have updated the component with ListViewSimplified.

And @Zangetsu101 will update the  SideNav and TopNav wrapper for form config.

**Review by euanmillar — APPROVED** [review]

**Pull request merged**